20260218-3files/AGENTS.md

```
@@ -0,0 +1,90 @@
+- **Build**: kotlinc 직접 컴파일 또는 Gradle
+- **Platform**: macOS / Linux (ANSI 터미널 필요)
+
+## Architecture
+
+```
+┌──────────┐     ┌──────────┐     ┌──────────┐
+│  Input   │────▶│   Game   │────▶│ Renderer │
+│(키보드)   │     │ (루프)    │     │(콘솔출력) │
+└──────────┘     └──────────┘     └──────────┘
+                      │
+              ┌───────┼───────┐
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)